proto/babylon/btcstaking/v1/btcstaking.proto

```diff
@@ -109,6 +109,19 @@ message BTCDelegation {
     uint32 params_version = 16;
 }
 
+// DelegatorUnbondingInfo contains the information about transaction which spent
+// the staking output. It contains:
+// - spend_stake_tx: the transaction which spent the staking output
+// - spend_stake_tx_inclusion_block_hash: the block hash of the block in which
+// spend_stake_tx was included
+// - spend_stake_tx_sig_inclusion_index: the index of spend_stake_tx in the block
+message DelegatorUnbondingInfo {
+    // spend_stake_tx is the transaction which spent the staking output. It is
+    // filled only if spend_stake_tx is different than unbonding_tx registered
+    // on the Babylon chain.
+    bytes spend_stake_tx = 1;
+}
```

Comment:
Do we still need this struct given that it only contains 1 field?

Reply:
yup, this signals whether unbonding actually happened i.e
```
if del.BtcUndelegation.DelegatorUnbondingInfo != nil
```
delegation is considered unbonded.

And if 

```
len(del.BtcUndelegation.DelegatorUnbondingInfo.SpendStakeTx) > 0 
```

then unbonding happened through unexpected unbonding transaction i.e the transaction different that `del.BtcUndelegation.UnbondingTx`

Assuming most of unbondings will happen through unbonding transaction committed to Babylon, this modelling enables us to save space i.e we do not store stake spending transaction if it happens to be already known unbonding transaction.

Reply:
OK sounds good. Probably we can wrap some functions for `BTCDelegation` to signal these in the future :+1: